rollup.config.js

```diff
@@ -5,7 +5,7 @@ import dts from "rollup-plugin-dts";
 import { terser } from "rollup-plugin-terser";
 import peerDepsExternal from "rollup-plugin-peer-deps-external";
 import json from "@rollup/plugin-json";
-import replace from '@rollup/plugin-replace';

```

Comment:
Add this prettier config (pinned to prettier `=2.0.5`) to the project and add a package.json rule for it, please:
```javascript
module.exports = {
  trailingComma: 'es5',
  tabWidth: 2,
  semi: true,
  singleQuote: true,
  printWidth: 120,
};
```

Reply:
done